.github/workflows/ci.yml

```
@@ -322,6 +322,14 @@ jobs:
     - name: Run testing against the Git checkout
       run: npm test
 
+    - name: Produce coverage report
+      run: npm run coverage
+    - name: Upload coverage data
+      uses: codecov/codecov-action@v2
+      with:
+        name: ${{ matrix.name }}
+        files: out/coverage.lcov
```

Let's flag each report. This way we can see which code was executed on each platform.
```suggestion
        files: out/coverage.lcov
        flags: >-
          GHA,
          ${{ runner.os }},
          ${{ matrix.os }},
          ${{ matrix.node-version }}
```